hexa/workspace/apps.py

```diff
@@ -0,0 +1,6 @@
+from hexa.app import CoreAppConfig
+
+
+class VisualizationsConfig(CoreAppConfig):
+    name = "hexa.workspace"
```

Comment:
I'd prefer in plural

Reply:
Okay

> You have to create the relations on the model too. Check on the `Team` model for an example.
> 
> You also have to implement the graphql part of that as well.

Oh okay. I was referring to the task desription, but okay I'll add update it.